Gpufit/python/pygpufit/gpufit.py

```diff
@@ -238,41 +251,45 @@ def fit_constrained(data, weights, model_id, initial_parameters, constraints=Non
     chi_squares = np.zeros(number_fits, dtype=np.float32)
     number_iterations = np.zeros(number_fits, dtype=np.int32)
 
+    # argtypes are grabbed from gpufit_func, but are the same for cpufit_func
+    atypes = gpufit_func.argtypes
```

Comment:
just cleaning up the name so future devs don't think the argtypes are specific to gpufit